registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/java/io/mosip/registration/processor/workflow/manager/WorkflowManagerApplication.java

```diff
@@ -0,0 +1,27 @@
+package io.mosip.registration.processor.workflow.manager;
+
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import io.mosip.registration.processor.workflow.manager.stage.WorkflowActionApi;

```

Comment:
can you make workflowmanager as a single folder and remove stage from folder, since this is part of workflow engine and not part of any stage.

Reply:
fixed